test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -145,6 +145,86 @@ public async Task start_displays_error_on_invalid_function_json()
             }, _output);
         }
 
+        [Fact]
+        public async Task start_displays_error_on_invalid_host_json()
+        {
+            var functionName = "HttpTriggerCSharp";
+
+            await CliTester.Run(new RunConfiguration[]
+            {
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        "init . --worker-runtime dotnet",
+                        $"new --template Httptrigger --name {functionName}",
+                    },
+                    Test = async (workingDir, _) =>
+                    {
+                        var filePath = Path.Combine(workingDir, "host.json");
+                        string hostJsonContent = "{ \"version\": \"2.0\", \"extensionBundle\": { \"id\": \"Microsoft.Azure.Functions.ExtensionBundle\", \"version\": \"[1.*, 2.0.0)\" }}";
+                        await File.WriteAllTextAsync(filePath, hostJsonContent);
+                    }
+                },
+                new RunConfiguration
+                {
+                    Commands = new[]
+                    {
+                        "start"
+                    },
+                    ExpectExit = true,
+                    ExitInError = true,
+                    ErrorContains = new[] {     "Extension bundle configuration should not be present" },
+                    Test = async (_, p) =>
+                    {
+                        // give the host time to load functions and print any errors
+                        await Task.Delay(TimeSpan.FromSeconds(10));
+                        p.Kill();
+                    }
+                }
+            }, _output);
+        }
+
+        [Fact]
+        public async Task start_displays_error_on_missing_host_json()
```

Comment:
Move tests to use theory similar to : https://github.com/Azure/azure-functions-host/blob/aa3d0576d5bc2f73a86bdb4c32af3c97be9b9f14/test/WebJobs.Script.Tests/Configuration/HttpWorkerOptionsSetupTests.cs#L76

Also, add test for non dotnet languages and bundles should be allowed.

Reply:
The tests for non dotnet languages are already present. Will double check to confirm

Reply:
Can't move to theory since there is only a single data point to test